### ISSUE TITLE
ACOS strip dynamic value from CLI

### ISSUE THREAD

**Issue by broxio** [ytti/oxidized#446]
From below command will give output with dynamic value.

```
#show version
      Hard Disk primary image version 2.7.1-P3-76-1, build 42840
      Hard Disk secondary image (default) version 2.7.1-P5-SP2, build 3
      Compact Flash primary image (default) version 2.7.1-P2, build 57
      Compact Flash secondary image version 2.7.1-P2, build 57
      Last configuration saved at May-26-2016, 17:21
      Hardware: 16 CPUs(Stepping 7), Single 97G Hard disk
      Memory 65171 Mbyte, Free Memory 47709 Mbyte
      Hardware Manufacturing Code: 133011
      Current time is May-25-2016, 13:47
      The system has been up 111 days, 22 hours, 38 minutes
```
```
#show running-config
!Current configuration: 429923 bytes
!Configuration last updated at 17:21:52 PST Thu May 25 2016
!Configuration last saved at 17:21:52 PST Thu May 25 2016
!Configuration last synchronized at 17:22:01 PST Thu May 25 2016
!version 2.7.1-P5-SP2, build 3 (Jun-09-2014,21:05)
```

also add #show bootimage 

I had to strip those value out otherwise every time poller start, the config will always have a new version.

cc2c47f

**Comment by ytti** [comment]
Can you send pull request please?

**Comment by broxio** [comment]
closed PR #447
